<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxOSRadioGroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>281</width>
      <height>67</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButtonWindows">
       <property name="text">
        <string>Windows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonLion">
       <property name="text">
        <string>OS X Lion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonLinux">
       <property name="text">
        <string>Linux</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxOSCheckBoxGroup">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>282</width>
      <height>67</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBoxWindows">
       <property name="text">
        <string>Windows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxLion">
       <property name="text">
        <string>OS X Lion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxLinux">
       <property name="text">
        <string>Linux</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 0, 0);</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_evenements">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>évènements</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserAffichage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelCode">
       <property name="text">
        <string>Entrez votre code : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCode">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(114, 159, 207);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>169</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelChoix">
       <property name="text">
        <string>Choix</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_OS">
       <item>
        <property name="text">
         <string>Linux</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Windows</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OS X Lion</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>254</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButtonLinux">
       <property name="text">
        <string>Linux</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWindows">
       <property name="text">
        <string>Windows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLion">
       <property name="text">
        <string>OS X Lion</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLangages">
    <property name="title">
     <string>Langages</string>
    </property>
    <widget class="QMenu" name="menuWeb">
     <property name="title">
      <string>Web</string>
     </property>
     <addaction name="actionPHP"/>
     <addaction name="actionJavascript"/>
    </widget>
    <addaction name="actionC"/>
    <addaction name="separator"/>
    <addaction name="actionJAVA"/>
    <addaction name="actionCpp"/>
    <addaction name="separator"/>
    <addaction name="menuWeb"/>
   </widget>
   <addaction name="menuLangages"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionC">
   <property name="text">
    <string>C</string>
   </property>
  </action>
  <action name="actionJAVA">
   <property name="text">
    <string>JAVA</string>
   </property>
  </action>
  <action name="actionCpp">
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionPHP">
   <property name="text">
    <string>PHP</string>
   </property>
  </action>
  <action name="actionJavascript">
   <property name="text">
    <string>Javascript</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
